onnxruntime/test/onnx/microbenchmark/resize.cc

```diff
@@ -0,0 +1,60 @@
+#include <limits>
+
+#include "common.h"
+
+#include <benchmark/benchmark.h>
+#include "core/common/safeint.h"
+#include "core/framework/allocator.h"
+#include "core/mlas/lib/mlasi.h"
+#include "core/providers/cpu/tensor/upsample.h"
+#include "core/util/math_cpuonly.h"
+#include "core/util/qmath.h"
+#include "core/util/thread_utils.h"
+
+using namespace onnxruntime;
+
+static void BM_NhwcUpsampleBilinear(benchmark::State& state) {
+  const int64_t output_height = static_cast<int64_t>(state.range(0));
+  const int64_t output_width = static_cast<int64_t>(state.range(1));
+  constexpr int64_t batch_size = 1;
+  constexpr int64_t num_channels = 256;
+  constexpr int64_t input_height = 32;
+  constexpr int64_t input_width = 32;
+  const int64_t height_scale = output_height / input_height;
+  const int64_t width_scale = output_width / input_width;
+  const std::vector<float> roi{0.0f, 0.0f, 0.0f, 0.0f, 1.0f, 1.0f, 1.0f, 1.0f};
+  constexpr bool use_extrapolation = false;
+  constexpr float extrapolation_value = 0;
+  constexpr size_t XdataBaseSize = batch_size * num_channels * input_height * input_width;
+  const float* const XdataBase = GenerateArrayWithRandomValue<float>(XdataBaseSize, std::numeric_limits<float>::min(), std::numeric_limits<float>::max());
```

Comment:
Could you please also benchmark uint8 and int8? It is one of our main target.

Reply:
Added